src/views/css/main.ts

```diff
@@ -64,7 +64,7 @@ const promptGrid = {
     },
 };
 
-const sessionsHeight = `(100vh - ${titleBarHeight + infoPanelHeight}px)`;
+const sessionsHeight = `(100vh - ${titleBarHeight + infoPanelHeight + 5}px)`;
```

Comment:
I had to comment the reason. I noticed one thing and it was that when I added to the tab bars height, at scroll you could have seen like 3px of overflow of the content into the tab bar. After playing with the whole thing for a little while, this was the holly number that saved the calculations